src/components/Board.js

```diff
@@ -1,33 +1,125 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import axios from 'axios';
+import emoji from 'emoji-dictionary';
+
 
 import './Board.css';
 import Card from './Card';
 import NewCardForm from './NewCardForm';
 import CARD_DATA from '../data/card-data.json';
 
+
 class Board extends Component {
-  constructor() {
-    super();
-
-    this.state = {
-      cards: [],
-    };
-  }
-
-  render() {
-    return (
-      <div>
-        Board
-      </div>
-    )
-  }
 
-}
+	static propTypes = {
+		updateStatusCallback: PropTypes.func,
+		url: PropTypes.string,
+		name: PropTypes.string
+	}
+	constructor() {
+		super();
+
+		this.state = {
+			cards: [],
+		};
+
+	}
+
+
+
+	componentDidMount(){
+		axios.get(`${this.props.url}/${this.props.name}/cards`)
+		.then((response) =>{
+			this.props.updateStatusCallback("Successfully loaded board", "success")
+
+			this.setState({
+				cards: response.data
+			})
+		})
+		.catch((error) => {
+			this.props.updateStatusCallback(`Failed to load: ${error.message}`, "failure")
+		});
+
+	}
+
+	componentDidUpdate(prop, state) {
+		if (prop.name !== this.props.name) {
+			axios.get(`${this.props.url}/${prop.name}/cards`)
+			.then((response) =>{
+				this.props.updateStatusCallback("Successfully loaded board", "success")
 
-Board.propTypes = {
+				this.setState({
+					cards: response.data,
+					name: prop.name
+				})
+			})
+			.catch((error) => {
+				this.props.updateStatusCallback(`Failed to load: ${error.message}`, "failure")
+			});
+		}
+	}
+
+	addCard = (card) => {
+		card.emoji = emoji.getName(card.emoji)
+
+		axios.post(`${this.props.url}/${this.props.name}/cards`, card)
+
+		.then((response) => {
+			let updatedCards = this.state.cards;
+			updatedCards.push({card});
```

Comment:
You're adding the card, but not using the ID from the API. 

Instead you should do:

```JavaScript
card.id = response.data.card.id
```